core/sentry.js

```diff
@@ -0,0 +1,112 @@
+/* @flow */
+
+const Raven = require('raven')
+const logger = require('./logger')
+const log = logger({
+  component: 'Sentry'
+})
+
+const { omit } = require('lodash')
```

Comment:
`const _ = require('lodash')`? (less changes in future patches in case we use more functions)

Reply:
I would apply the rule of 3 (if we start using more than 2 functions from lodash, better to import the whole module) but it's true I have a weird illogical dislike of the `_.xxxxx` syntax

Reply:
Prefixing functions with the module/namespace they're coming from is considered good practice in at least 3 languages I know of: Go, Elm and Clojure.
So I was assuming it would be a good idea to do the same in JS :P

Reply:
Knowing lodash myself, I would have guessed an unprefixed `omit(...)` function call would be lodash-related. But knowing is always more comfortable than guessing :see_no_evil: